docker/Dockerfile.centos.7

```diff
@@ -62,16 +62,16 @@ RUN yum -y install                                                 \
            libiscsi-devel numactl-devel graphviz                   \
            libyaml-devel                                           \
            scons pylint file ShellCheck python-pep8 lcov python    \
-           python2-pygithub yum-plugin-copr
+           python2-pygithub yum-plugin-copr hwloc-devel            \
+           openmpi3-devel environment-modules
```

Comment:
It doesn't hurt, but I suspect specifying `environment-modules` here is superfluous.  `openmpi3` should bring it in and `openmpi3-devel` will bring `openmpi3` in.